tasks/managed/push-rpm-to-koji/tests/test-push-rpm-to-koji.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-rpm-to-koji
+spec:
+  description: |
+    Test the push-rpm-to-koji task.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:latest
+            script: |
+              if [ ! -f "$(workspaces.data.path)/data" ] ; then
```

Comment:
not needed

Reply:
Ack, good to know. I tested in my workspace with a PVC, so ever time I need to clean up.

Is there a way or doc to mention testing the pipeline in local?

Reply:
Thanks for sharing.